easybuild/easyconfigs/b/binutils/binutils-2.25-GCC-4.9.2.eb

```diff
@@ -11,12 +11,15 @@ source_urls = [GNU_SOURCE]
 
 toolchain = {'name': 'GCC', 'version': '4.9.2'}
 
-dependencies = [
-    ('zlib', '1.2.8'),
-]
+dependencies = [('zlib', '1.2.8')]
+
+builddependencies = [('binutils', '2.25', '', True)]
```

Comment:
this doesn't work (as you pointed out), since this implies that both `zlib/1.2.8` and `zlib/1.2.8-GCC-4.9.2` need to be loaded at the same time

moreover, I don't think that the binutils being used to build binutils matters a lot?

unless compiling binutils involves the compiler generating AVX2 instructions, which can be a problem on RHEL6 with the OS-provided binutils (cfr. one of the @hpcleuven systems)